qt/widgets/mplcpp/test/FigureCanvasQtTest.h

```diff
@@ -41,6 +41,8 @@ class FigureCanvasQtTest : public CxxTest::TestSuite {
   void testToDataCoordinatesReturnsExpectedPoint() {
     FigureCanvasQt canvas{111};
     canvas.gca().plot({1, 2, 3, 4, 5}, {1, 2, 3, 4, 5});
+    canvas.draw(); // We now need to draw as 3.2 changed when autoscaling occurs in plots, to only when drawn, not
```

Comment:
Should there be a version control here? Presumably we don't need this for version 3.1.3?

Reply:
We don't need it, effectively we have no need to check versions as the behaviour externally is very similar